src/main/frontend/extensions/code.css

```diff
@@ -21,52 +21,36 @@
   }
 
   &-calc {
-    @apply absolute right-0 text-sm;
-    padding: 0 0.25em;
-    margin-top: 24px;
-    top: 3px;
-    z-index: 1;
-    background: transparent;
+    @apply text-sm;
+    padding: 0.25em;
     width: max-content;
     text-align: right;
 
     &-output-line {
-      height: 23px;
-      display: flex;
-      flex-direction: column;
-      justify-content: space-around;
-
-      > span {
-          font-family: Fira Code, Monaco, Menlo, Consolas, 'COURIER NEW', monospace;
-      }
+      height: 1.7em;
+      word-break: normal;
```

Comment:
A special case for word-break:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/72891/153363913-05ff7108-1b85-439d-815d-585d759046f7.png">

Reply:
Replaced with `white-space: pre`:
![CleanShot 2022-02-11 at 15 10 03](https://user-images.githubusercontent.com/479169/153550697-53862a1f-a614-48db-9245-0f4f916cc9a8.png)